docs/src/environments.md

```diff
@@ -5,6 +5,8 @@ The following discusses Pkg's interaction with environments. For more on the rol
 ## Creating your own projects
 
 So far we have added packages to the default project at `~/.julia/environments/v1.6`. It is however easy to create other, independent, projects.
+This approach has the benefit of allowing you to check-in a Project.toml (and even a Manifest.toml if you wish) into version control (e.g. git)
+to ensure that the code you're version controlling also has a reproducible environment.
```

Comment:
Maybe it's just the way I'm reading it, but to me, this "if you wish" language seems to imply that if you check the `Project.toml` file into source control, you get a reproducible environment, even if you choose to not check in the `Manifest.toml`. Perhaps this sentence should be reworded to make it clear that you only get reproducible environments if the `Manifest.toml` file is also checked in.

Reply:
Well, a Project.toml alone with the typical type of compat entries should give a reproducible environment in a semver sense. I think the concept of reproducibility is a spectrum and I'm not sure we want to promote checking in manifests too much?

Reply:
You could call that a "semver-compatible" environment, but I definitely wouldn't call that a reproducible environment. If you don't get the exact same package versions, then I don't think that's a reproducible environment.

I agree that we don't want to encourage people to check in manifests for packages (only for applications). So I would prefer that we just remove this sentence entirely.

Reply:
Fair point

```suggestion
```